src-tauri/src/main.rs

```diff
@@ -329,19 +328,6 @@ async fn install_northstar_caller(
     }
 }
 
-/// Update Northstar install in the given path
-#[tauri::command]
-async fn update_northstar(
```

Comment:
TIL that we never called this function in frontend, whoops.

Removal of that function should be separate though IMO.

(I'm running on very little time atm, so the smaller the changes, the larger the chance of merging in reasonable time :P)